source/directives/clickable.rst

```diff
@@ -0,0 +1,232 @@
+
+    For example:
+
+    .. code-block:: rst
+
+       .. clickablearea:: question2
+          :question: Click the rainbow color(s)
+          :table:
+          :correct: 1,0;2,2;3,1;3,3;4,2
+          :incorrect: 2,1;2,3;3,2;4,1;4,3
+
+          +-------+---------+--------+
+          |  Red  |  Orange | Yellow |
+          +-------+---------+--------+
+          | White |  Green  | White  |
+          +-------+---------+--------+
+          |  Blue |  White  | Indigo |
+          +-------+---------+--------+
+          | White |  Violet | White  |
+          +-------+---------+--------+
+
+    Table location numbering begins at 1.
+    When referring to table cells, the upper left cell ('Red' in this example) is location ``1, 1``.
+    The bottom right cell is at location ``4, 3``.
+
+correct
+    ``Formatted List``. A list of table coordinates that are marked as correct when clicked.
+
+
+incorrect
+    ``Formatted List``. A list of table coordinates that are marked as correct when clicked.
+
+For both of the formatted lists, the list is built of cell coordinates.
+
+- The first number is the table row.
+- The second number is the table column.
+
+  A column coordinate equal to ``0`` means all columns are selected.
+
+- Each table coordinate is separated with ``;``.
+- No white space is allowed.
+
+Optional Arguments
+------------------
+
+feedback
+    ``String``. Define incorrect feedback displayed when the **Check Me** button is pressed.
+
+    The default is:
+    
+    .. code-block:: none
+
+        Incorrect. You clicked on X of the Y correct elements and A of the B incorrect elements.
+
+    The feedback for correct responses is always::
+
+        You are Correct!
+
+
+Languages supported
+-------------------
+
+The ``clickablearea`` directive is language agnostic.
+Nothing is actually executed or interpreted.
+It is up to the author to ensure the syntax and grammar within a clickable area
+makes sense - no syntax checking is performed.
+
+Sphinx configuration options
+----------------------------
+
+No directive specific configuration options exist.
+
+Internationalization
+....................
+
+tbd.
+
+Known limitations
+-----------------
+
+No syntax highlighting for ``:iscode:`` examples.
+
+The text for all options (question, feedback, etc.) must appear on a single line.
+No newlines are allowed.
+
+There is no way to suppress carriage returns in the source,
+which can make for long lines of text.
+If your text editor wraps, this can make the source difficult to read.
+
+There is no way to set the width or alignment of a table rendered using ``:table:``.
+Columns widths will automatically adjust to fill the available space.
+
+Examples
+--------
+A simple table example.
+
+.. tabbed:: example1
+
+   .. tab:: Source
+
+      .. literalinclude:: clickable_examples/ca-ex1.txt
+         :language: rst
+
+   .. tab:: Try It
+
+      .. include:: clickable_examples/ca-ex1.txt
```

Comment:
These are not working, it is only re-showing the code rather than a working clickable example.

Reply:
I'm not seeing what you are seeing.
When I build locally, or publish a temp copy here:
https://daveparillo.github.io/runestoneinteractive.github.io/build/html/directives/clickable.html#examples
it renders fine.
Thoughts?

Reply:
Well, when I look at what you have committed it looks great.  But, I rebuilt it with the dev version of runestone and that is when things go funny. Which means I must have a bug in the components.  I'll have to look into that as a separate issue from this.